prelude/prelude.purs

```diff
@@ -913,6 +1056,8 @@ module Control.Monad.Eff where
 
   type Pure a = forall e. Eff e a
 
+  -- | Run an effectful computation with no side effects, either because there 
+  -- | are none or because all possible side effects are handled.
```

Comment:
I don't think this is a case of "either". The key is that the argument to `runPure` can be run in any context, because it doesn't require any fixed effects.